### PR TITLE
Set up the octo summit cluster 1 for CR deployment

### PR DESCRIPTION
**Pull request by harshad16**
Set up the octo summit cluster 1 for CR deployment
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

**Comment by HumairAK** [comment]
@harshad16 you'll need an app to deploy cluster resources: https://github.com/operate-first/apps/blob/master/argocd/overlays/moc-infra/applications/envs/moc/smaug/cluster-management/cluster-resources.yaml

**Comment by harshad16** [comment]
/unhold
Ready for review

File: argocd/overlays/moc-infra/applications/envs/octo/gohan/cluster-resources.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: cluster-resources
+  annotations:
+    notifications.argoproj.io/subscribe.on-sync-succeeded.slack: "argocd-notifications"
+    notifications.argoproj.io/subscribe.on-sync-running.slack: "argocd-notifications"
+    notifications.argoproj.io/subscribe.on-sync-failed.slack: "argocd-notifications"
+    notifications.argoproj.io/subscribe.on-sync-status-unknown.slack: "argocd-notifications"
+    notifications.argoproj.io/subscribe.on-sync-degraded.slack: "argocd-notifications"
```
Comment (HumairAK) [inline argocd/overlays/moc-infra/applications/envs/octo/gohan/cluster-resources.yaml:10]: can you remove these, this enables argocd notifications for this app in slack and I don't think we're ready to enable these for other apps.

Comment (harshad16) [reply argocd/overlays/moc-infra/applications/envs/octo/gohan/cluster-resources.yaml]: sounds good 
removed :+1:

**Comment by harshad16** [comment]
Ready for review again

**Comment by HumairAK** [comment]
@harshad16 I've approved it, and put it on hold so you can unhold it for merge whenever you are available so you can verify changes.

**Pull request merged**